.github/workflows/ci.yaml

```
@@ -16,29 +17,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
+      - uses: dtolnay/rust-toolchain@stable
       - run: rustup component add rustfmt
```

The ideal way here would be to use input parameters `components` and `toolchain`.

```suggestion
        with:
          toolchain: stable
          components: clippy, rustfmt
```